parachain/primitives/beacon/src/lib.rs

```diff
@@ -5,9 +5,11 @@
 pub mod bits;
 pub mod bls;
 pub mod config;
+pub mod deneb;
```

Comment:
I think the module layout can be improved. Its confusing to add a `deneb` module at the top level. It should be a child of `types`.

For example:
```rust
use crate::types::capella::ExecutionPayloadHeader
use crate::types::deneb::ExecutionPayloadHeader
```

Reply:
Reorganized in https://github.com/Snowfork/snowbridge/pull/981/commits/3580c4de7d6400880f3a4df171ffb42b5000737b